examples/lighting-app/silabs/data_model/lighting-thread-app.zap

```diff
@@ -1509,6 +1509,14 @@
               "source": "client",
               "isIncoming": 1,
               "isEnabled": 1
+            },
+            {
+              "name": "RetrieveLogsResponse",
```

Comment:
Diagnostic logs are not currently supported on Silabs platform. since this zap config is specific to the Silabs platform we should remove this cluster from it.

The other zap files are common zap config. We currently have to live it it enabled but not this one